savepoint-tanstack/src/styles.css

```
@@ -19,86 +19,80 @@
-  --font-cartridge-display: "Bowlby One";
-  --font-cartridge-sans: "Archivo";
-  --font-cartridge-mono: "Space Mono";
-  --font-aurora-display: "Space Grotesk";
-  --font-aurora-mono: "JetBrains Mono";
 }
 
-@custom-variant dark (&:is(.dark *, .y2k *, .jewel *, .aurora *));
-@custom-variant y2k (&:is(.y2k *));
-@custom-variant jewel (&:is(.jewel *));
-@custom-variant aurora (&:is(.aurora *));
+@custom-variant dark (&:is(.dark *));
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@custom-variant`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)